### ISSUE TITLE
Compile tetris.v

### ISSUE THREAD

**Issue by k1attila12** [vlang/v#5980]
**V version:**  0.1.28 6fcc6cd + gcc 8.1.0 x86_64
**OS:**  win10

**What did you do?** compile tetris.v (v tetris.v or v run tetris.v)
  All other non graphic sample is compiled perfectly.

**What did you expect to see?** compile tetirs.v

 
**What did you see instead?**
```
Unable to find vs installation (attempted to use lib path "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.11.25503 \lib\X64")
==================
C:\Users\fiskus\AppData\Local\Temp\ccBV2oQD.o:tetris.tmp.c:(.rdata$.refptr.IID_ID3D11Texture2D[.refptr.IID_ID3D11Texture2D]+0x0): undefined reference to `IID_ID3D11Texture2D'
collect2.exe: error: ld returned 1 exit status
...
==================
(Use `v -cg` to print the entire error message)

[1m[31mbuilder error[39m[22m: 
==================
C error. This should never happen.

If you were not working with C interop, please raise an issue on GitHub:

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang
```
freetype is installed perfectly.
i recompiled v with make.

Thank you in advance Attila

**Comment by nedpals** [comment]
This must be a Sokol/gg-related issue hmmm

**Comment by medvednikov** [comment]
does `v hello.v` work?

**Comment by k1attila12** [comment]
Everythin g works well, except graphic.

**Comment by JalonSolov** [comment]
GUI code has been failing to compile for several weeks.  There are reports on Discord from at least July 9th.  There is a previous issue about tetris.v failing to compile from a week ago (#5881).

**Comment by medvednikov** [comment]
@JalonSolov on Windows only.

**Comment by JalonSolov** [comment]
Yes, and his report was one Windows.

**Comment by JalonSolov** [comment]
Fixed with PR #6155.
